bids/layout/layout.py

```diff
@@ -525,14 +554,17 @@ def parse_file_entities(self, filename, scope='all', entities=None,
         return parse_file_entities(filename, entities, config,
                                    include_unmatched)
 
-    def add_derivatives(self, path, **kwargs):
+    def add_derivatives(self, path, create_derivative_database_files=False, **kwargs):
```

Comment:
I was working on that already today!  The test found an unrelated bug related to not persisting the Configs's to the sqldatabase at all.  This will be coming shortly.